packages/manager/modules/carbon-calculator/src/index.scss

```diff
@@ -3,3 +3,12 @@
 @import 'bootstrap4/scss/_mixins.scss';
 @import 'bootstrap4/scss/_utilities.scss';
 @import 'bootstrap4/scss/_grid.scss';
+
+.carbon-balance {
+  padding-bottom: 2em;
+
+  img {
+    padding-bottom: 1em;
+    border-bottom: 1px solid #d3d3d3;
+  }
```

Comment:
**nitpick:** Avoid custom css and use bootstrap utility classes to set padding and border

Reply:
fixed !